.github/workflows/release.yaml

```diff
@@ -24,7 +24,10 @@ jobs:
         run: echo ::set-output name=raw_version::"${GITHUB_REF#refs/*/v}"
       - name: Get release information
         id: changes
-        run: echo ::set-output name=release_notes::"$(go run ./util/cmd/print_changes ${{ steps.tag.outputs.version }})"
+        # Need to escape new line characters in order to get multiline release notes.
+        run: |
+          notes=$(go run ./util/cmd/print_changes ${{ steps.tag.outputs.version }})
+          echo ::set-output name=release_notes::"${notes//$'\n'/'%0A'}"
```

Comment:
nice shell foo!
For my reference:
https://tldp.org/LDP/abs/html/string-manipulation.html
http://www.gnu.org/software/bash/manual/html_node/ANSI_002dC-Quoting.html